spec/controllers/miq_policy_controller/rsop_spec.rb

```diff
@@ -0,0 +1,28 @@
+include UiConstants
```

Comment:
@h-kataria @dclarizio I think we need to not add globals - neither in tests nor code.